src/release-it-config/README.md

```diff
@@ -0,0 +1,123 @@
+# Silvermine-release
+
+If you have this project installed globally you can use the following commands:
+
+
+## Run a Release
+
+With a clean working directory, run:
+
+`silvermine-release --release`
+
+Begins the release process for a full release:
+
+   * Generates a changelog of bugfixes and features.
+   * Calculates the next semver version number and bumps the version in `package.json`.
+   * Commits the changes.
+
+## Run a Prerelease
+
+With a clean working directory, run:
+
+`silvermine-release --pre-release=rc`
+
+...or an alpha:
+
+`silvermine-release --pre-release=alpha`
+
+Begins the release process for a pre-release version:
+
+   * Calculates the next semver version number and bumps the version in `package.json`.
+   * Commits the changes.
+
+## Tag the Release
+
+With a clean working directory, run:
+
+`silvermine-release --tag`
+
+Creates git tags for publishing a release:
+
+   * Creates the local tag based on the current version found in
+   package.json.
+   * Pushes the tag to your repository.
+
+## NPM Scripts
+
+To avoid having to install the global package, you can also add npm scripts
+to your `package.json`'s `scripts: {}` object.
+
+For example:
+
+```json
+   "release-rc": "silvermine-release --pre-release=rc",
+   "release": "silvermine-release"
```

Comment:
ADIRL, the default is no longer the "help" command and running a release uses a `--release` flag